canonical-web-and-design/ubuntu.com
# Rebuild /telco

## Done

- Rebuild /telco page 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/telco
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the page against the [copy doc](https://docs.google.com/document/d/15FY_IaT39V81FQcoSoEJpmRRjBX80PJ7uxjgzHnDDws/edit)
- Check all the links link to correct pages
- Check design


## Issue / Card

Fixes [#3231](https://github.com/canonical-web-and-design/web-squad/issues/3231)
